src/test/java/org/jfrog/hudson/pipeline/integration/MockServer.java

```diff
@@ -0,0 +1,31 @@
+package org.jfrog.hudson.pipeline.integration;
+
+import org.mockserver.integration.ClientAndServer;
+import org.mockserver.model.HttpRequest;
+import org.mockserver.model.HttpResponse;
+
+
+/**
+ * @author Alexei Vainshtein
+ */
+
+public class MockServer {
```

Comment:
Nice and simple mock! But please remove `public` access modifier from the class and functions.

Reply:
Done